jetty-websocket/javax-websocket-server/src/main/java/org/eclipse/jetty/websocket/javax/server/JavaxWebSocketServerContainer.java

```diff
@@ -71,34 +80,86 @@ public static WebSocketContainer getWebSocketContainer()
         if (!(handler instanceof ServletContextHandler))
             return null;
 
-        return (javax.websocket.WebSocketContainer)handler.getServletContext().getAttribute("javax.websocket.server.ServerContainer");
+        return (javax.websocket.WebSocketContainer)handler.getServletContext().getAttribute(ServerContainer.class.getName());
```

Comment:
Use `"javax.websocket.server.ServerContainer"` not `ServerContainer.class.getName()`.
People do search for this critical term.